junit-to-json/src/lib/index.ts

```diff
@@ -10,9 +10,18 @@ function isTestCases(
 }
 
 export function processXmlResult(xmlContent: Buffer): ExercismTestRunnerResult {
-  const $ = load(xmlContent, { xmlMode: true })
+  const $ = load(xmlContent, {xmlMode: true})
   const parsed = parseTestSuites($)
 
+  if (parsed.length === 0) {
+    return {
+      version: 2,
+      tests: [],
+      status: "error",
+      message: "Unit test run did not produce any results. Did the tests die?"
```

Comment:
```suggestion
      message: "Unit test run did not produce any results. Did the test suite fail unexpectedly?"
```

Reply:
It returns a 0, same as a successful run. It's not even in PHPUnit's control. Exit in PHP just kills the run and you get a 0 exit code. That's why I went after the blank output.

As far as the message, the intent was to prompt the student to look for die in the code since that's what was causing this problem, but I'm good without whatever you'd like for the messaging. 

With the student I was working with on the exercise that prompted this update, they knew about die being in the code, but they submitted because exercism told them the tests passed. So I wonder if the failing message is enough or if we should bring attention to the potential exit or die call in the code that caused it to produce no output.

Reply:
Ah, interesting!

With that exit code in mind, yes I think this is the right approach for now.

As far as the message to show, maybe me can do "...did the tests finish completely?\n\nUsing the `die` function in your code will cause the test run to not produce any output."

Reply:
Sounds good - should I also mention exit?

Reply:
Maybe something like this if we put it all together?

```suggestion 
Test run did not produce any output. Check your code to see if the code exits unexpectedly before the report is generated.

E.g. Using the `die` function will cause the test runner to exit unexpectedly.
```

Reply:
I've updated and pushed up the change you suggested.